src/Data/Types/DecimalType.php

```diff
@@ -0,0 +1,47 @@
+<?php
+namespace LORIS\Data\Types;
+
+/**
+ * A DecimalType represents a data type which may have a fractional
+ * part to its representation.
+ *
+ * @license http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ */
+class DecimalType implements \LORIS\Data\Type
+{
+    /**
+     * Construct a DecimalType
+     */
+    public function __construct()
+    {
+    }
```

Comment:
why do some classes have empty constructors like this and others no constructors (like integer)

Reply:
Probably leftover from when I was first writing it, I'll remove the empty constructors.